package.json

```diff
@@ -17,7 +17,7 @@
     "build": "npm run clean && ngc -p tsconfig.json",
     "lint": "tslint src/**/*.ts",
     "clean": "del-cli lib '*.d.ts' '*.js' '*.metadata.json'",
-    "prepublish": "npm run build",
+    "prepare": "npm run build",
```

Comment:
Un-rename :)

Reply:
Prepublish is deprecated since NPM 5 and should be replaced by either prepare or prepublishOnly

Reply:
Cool, didn't know :). But just undo the change, I'm going to switch over to `ng-packagr` and will probably remove this step.